packages/@aws-cdk/aws-pipes-targets-alpha/test/integ.redshift.ts

```diff
@@ -0,0 +1,74 @@
+import { IPipe, ISource, Pipe, SourceConfig } from '@aws-cdk/aws-pipes-alpha';
+import { Cluster, ClusterType, NodeType } from '@aws-cdk/aws-redshift-alpha';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { RedshiftTarget } from '../lib/redshift';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-pipes-targets-redshift');
+const sourceQueue = new cdk.aws_sqs.Queue(stack, 'SourceQueue');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { restrictDefaultSecurityGroup: false });
+
+const sg = new ec2.SecurityGroup(stack, 'SecurityGroup', {
+  vpc,
+  allowAllOutbound: true,
+});
+// not recommended, only for integration test
+sg.addIngressRule(ec2.Peer.anyIpv4(), ec2.Port.tcp(5439), 'Ingress from Internet');
+
+const targetCluster = new Cluster(stack, 'MyCluster', {
+  clusterName: 'my-cluster',
+  masterUser: { masterUsername: 'admin' },
+  nodeType: NodeType.DC2_LARGE,
+  clusterType: ClusterType.SINGLE_NODE,
+  publiclyAccessible: true, // not recommended, only for integration test
```

Comment:
@redwheeler3 I assume the Redshift cluster needs to be publicly accessible in order for Pipes to hit it ?

Reply:
@redwheeler3, any insight on this one? I can get the pipe to execute (I see the `BatchExecuteStatement` API call in CloudTrail), but I don't see the query run in Redshift. I think my networking/security group look good but the Redshift cluster has to be publicly accessible for Pipes to hit it, right?

Reply:
Yes. For now, the Redshift cluster needs to be publicly accessible.